core/commonMain/src/kotlinx/io/bytes/BytesInput.kt

```diff
@@ -0,0 +1,30 @@
+package kotlinx.io.bytes
+
+import kotlinx.io.*
+import kotlinx.io.buffer.*
+
+/**
+ * In memory byte source created from [BytesOutput] or by using [buildInput].
+ *
+ * [BytesInput] isn't pooled so it's safe to throw it away.
```

Comment:
`throw it away` -> `not to close it`.
It's better to be explicit in the documentation